compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -817,6 +817,9 @@ error.invalid.init.invocation=\
 error.invalid.resource.function.invocation=\
   resource function can not be invoked with in a service
 
+error.invalid.function.pointer.invocation.with.type = \
+  function pointer with function type cannot be invoked
```

Comment:
suggestion: 
cannot invoke function pointer of type 'function'

Reply:
Changed.